src/app/helptext/data-protection/replication/replication.ts

```diff
@@ -108,8 +108,9 @@ export default {
 
   replicate_placeholder: T('Full Filesystem Replication'),
   replicate_tooltip: T('Completely replicate the selected dataset. The target \
-dataset will have all of the properties, snapshots, child datasets, and clones \
-from the source dataset.'),
+dataset will have all of the source dataset\'s properties, child datasets, \
+clones and snapshots that match the specified naming schema. Set \
+Snapshot Name Regular Expression to .* to replicate all snapshots.'),
```

Comment:
@themylogin So if I don't read the tooltip and don't set anything in either Regular Expression or naming schema, what would happen?